src/panels/lovelace/cards/hui-statistics-graph-card.ts

```diff
@@ -167,9 +223,22 @@ export class HuiStatisticsGraphCard extends LitElement implements LovelaceCard {
     this._getStatistics();
     // statistics are created every hour
     clearInterval(this._interval);
-    this._interval = window.setInterval(
-      () => this._getStatistics(),
-      this._intervalTimeout
+    if (!this._config?.energy_date_selection) {
```

Comment:
Don't we still need the interval with energy? Or does the energy collection emit events periodically?

Reply:
Energy collection self-schedules an hourly refresh. None of the other energy cards manage their own refresh lifecycle.